templates/common/config/stitches.config.ts

```diff
@@ -0,0 +1,14 @@
+import { createStitches } from '@stitches/react';
+import { customTheme } from '../src/theme/theme';
+import { globalTheme } from 'themes';
+
+export const {
```

Comment:
I would recommend we manage everything related to theme within the theme package. There is no need to create multiple themes. We just need to demonstrate how to extend the one you get from https://nextui.org/